appformer-client-api/src/main/java/org/appformer/client/context/DefaultEditorContextProviderImpl.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.appformer.client.context;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Alternative;
```

Comment:
This import is not used what is suspicious as it seems as we add implementation of interface that we will have more alternatives.

---
Maybe when will read rest of PR will find out I am wrong.

Reply:
This is the default implementation. We don't need an `@Alternative` there since this bean should be used when no alterantives to it are configured. Removed the import.